test/DynamicHelpTest.cs

```diff
@@ -0,0 +1,196 @@
+                "Get-Date", _.F1,
+                CheckThat(() => Assert.Equal(fullHelp, _mockedMethods.helpContentRendered)),
+                _.Enter
+                ));
+        }
+
+        [SkippableFact]
+        public void DynHelp_GetParameterHelp_And_Clear()
+        {
+            TestSetup(KeyMode.Cmd);
+            string emptyLine = new string(' ', _console.BufferWidth);
+
+            Test("Get-Date -Date", Keys(
+                "Get-Date -Date", _.Alt_h,
+                    CheckThat(() => AssertScreenIs(9,
+                        TokenClassification.Command, "Get-Date",
+                        TokenClassification.None, " ",
+                        TokenClassification.Parameter, "-Date",
+                        NextLine,
+                        emptyLine,
+                        TokenClassification.None, $"-Date <name>",
+                        NextLine,
+                        emptyLine,
+                        TokenClassification.None, "DESC: Specifies a date and time.",
+                        NextLine,
+                        TokenClassification.None, "Required: false, Position: 0, Default Value: None, Pipeline ",
+                        NextLine,
+                        "Input: True (ByPropertyName, ByValue), WildCard: false")),
+                _.Enter,
+                _.Enter,
+                    CheckThat(() => AssertScreenIs(1,
+                        TokenClassification.Command, "Get-Date",
+                        TokenClassification.None, " ",
+                        TokenClassification.Parameter, "-Date"))
+                ));
+        }
+
+        [SkippableFact]
+        public void DynHelp_GetParameterHelpMultiLine_And_Clear()
+        {
+            TestSetup(KeyMode.Cmd);
+            string emptyLine = new string(' ', _console.BufferWidth);
+
+            Test("Get-MultiLineHelp -OneAndHalf", Keys(
+                "Get-MultiLineHelp -OneAndHalf", _.Alt_h,
+                    CheckThat(() => AssertScreenIs(9,
+                        TokenClassification.Command, "Get-MultiLineHelp",
+                        TokenClassification.None, " ",
+                        TokenClassification.Parameter, "-OneAndHalf",
+                        NextLine,
+                        emptyLine,
+                        TokenClassification.None, $"-Date <name>",
+                        NextLine,
+                        emptyLine,
+                        TokenClassification.None, "DESC: Some very long description that is over the buffer width of ",
+                        TokenClassification.None, "60 characters but shorter than 120.",
+                        NextLine,
+                        TokenClassification.None, "Required: false, Position: 0, Default Value: None, Pipeline ",
+                        NextLine,
+                        "Input: True (ByPropertyName, ByValue), WildCard: false")),
+                _.Enter,
+                _.Enter,
+                    CheckThat(() => AssertScreenIs(1,
```

Comment:
Same here. Also, if any key would clear the dynamic help section, then maybe use a different key here, say `_.Escape`.

Reply:
Changed